comparison.md

```diff
@@ -169,7 +169,7 @@ Help improve our documentation! If you find anything wrong or which could be imp
 				<td><strong>Free SSL/HTTPs</strong></td>
 				<td>✅</td>
 				<td>❌️ Extra costs</td>
-				<td>❌️ Extra costs</td>
+				<td>⚠️ Provide own certificate via Self Serve SSL</td>
```

Comment:
The exact name Acquia uses for it is "self service SSL" but what's standing there would be enough to live with :).